tests/User Acceptance Testing Framework/scripts/setup_acceptance_test_environment/source_initiate-acceptance-tests.sh-pr-60

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+./scripts/banner "Preparing the Acceptance Test environment..."
+
+# If it is a randomized test
+
+if [ $1 -eq  $1 2>/dev/null ]; then
+  
+  TEST_NUMBER=$1
+
+fi
+
+# Preparing the environment
+
+mkdir $BMI_INSTANCE_DIR
+cd $BMI_INSTANCE_DIR
+rm -rf ims
+
+if [ ! -z  "`sudo ls /etc/ | grep redhat-release`" ]; then
+  cp /etc/hostname .
+  cat /etc/hostname | cut -f1 -d'.' > hostname
+  sudo cp hostname /etc
+fi
+
+git clone https://github.com/CCI-MOC/ims.git
+cd ims
+git fetch origin pull/60/head:pr-60
+git checkout pr-60
```

Comment:
okay, so this file is for testing out pr-60 which we have already merged. Does this have any other purpose? Can you put that in the file description on top.